simperium/core.py

```diff
@@ -309,6 +309,11 @@ def all(self, cv=None, data=False, username=False, most_recent=False, timeout=No
                 is supplied an empty list will be return if no updates are made
                 before the timeout is reached.
         """
+        try:
```

Comment:
To avoid reassigning the value of passed parameter maybe we could do something like this bellow:

```python
try:
    url += '&batch=%d' % int(batch)
except:
    url +='&batch=100'
```